src/Faker/Provider/Image.php

```diff
@@ -7,17 +7,15 @@
  */
 class Image extends Base
 {
-    protected static $categories = array(
-        'abstract', 'animals', 'business', 'cats', 'city', 'food', 'nightlife',
-        'fashion', 'people', 'nature', 'sports', 'technics', 'transport'
-    );
+    /** @var string */
+    private const BASE_URL = 'https://via.placeholder.com';
 
     /**
      * Generate the URL that will return a random image
      *
      * Set randomize to false to remove the random GET parameter at the end of the url.
      *
-     * @example 'http://lorempixel.com/640/480/?12345'
+     * @example 'http://via.placeholder.com/640x480.png?text=well+hi+there'
```

Comment:
That example doesn't match the output. What about this?
```suggestion
     * @example 'http://via.placeholder.com/640x480.png/CCCCCC?text=well+hi+there'
```